src/main/resources/webassets/js/custom/drive.js

```diff
@@ -0,0 +1,19 @@
+var base = base || {};
+base.driveController = (function() {
+    var controller = {
+        load: function() {
```

Comment:
Arrow function can be used (though watch for use of this in object method, won't bind properly)